spec/001-initialization.js

```diff
@@ -329,6 +329,66 @@ describe('ConstructorioID', function () {
       var session = new ConstructorioID();
       expect(session.on_node).to.be.false;
     });
+
+    describe('when used with different libraries on different subdomains', function () {
```

Comment:
What do we mean by "different libraries" here?

Reply:
I updated it but I still don't love it. Let me know if you can come up with a better one
2f13f6657691c5cd73d589fa4d98e225cde0f3b1